src/Illuminate/Log/Writer.php

```diff
@@ -262,6 +266,31 @@ public function useErrorLog($level = 'debug', $messageType = ErrorLogHandler::OP
     }
 
     /**
+     * Register a Gelf handler.
+     *
+     * @param  string  $host
+     * @param  int     $port
+     * @param  string  $protocol
+     * @param  string  $level
+     *
+     * @return \Psr\Log\LoggerInterface
+     */
+    public function useGelf($host, $port = 12201, $protocol = 'udp', $level = 'debug')
+    {
+        if ($protocol === 'udp') {
+            $transport = new UdpTransport($host, $port);
+        } elseif ($protocol === 'tcp') {
+            $transport = new TcpTransport($host, $port);
+        } else {
+            throw new \RuntimeException('Gelf transport supports only tcp and udp protocols. Invalid protocol passed.');
```

Comment:
@GrahamCampbell Thanks for notes. Fixed all, except this one. I don't get it. Import what?